react-time-track/src/services/user.js

```diff
@@ -1,6 +1,7 @@
 import { apiUrl } from "../util";
 
 const API_USERS = `${apiUrl}users`;
+//http://localhost:3000/api/users/1/projects
```

Comment:
I think is very useful to show a little comment in order to assimilate what your code does, thanks for that!